crates/tabby-common/src/config.rs

```diff
@@ -73,9 +73,13 @@ impl Default for ServerConfig {
     }
 }
 
+pub fn to_filename<S: AsRef<str>>(s: S) -> String {
```

Comment:
would it easier to read if we put `to_filename` a method of `RepositoryConfig`? e.g `name`.